src/core/core.controller.js

```diff
@@ -291,7 +291,7 @@ class Chart {
 			// the chart initialization but after setting basic chart / controller properties that
 			// can help to figure out that the chart is not valid (e.g chart.canvas !== null);
 			// https://github.com/chartjs/Chart.js/issues/2807
-			console.error("Failed to create chart: can't acquire context from the given item");
+			console.error("Failed to create chart: can't acquire context from the given item \n\n Tips: \n\n-Check if the dom element requested is indeed present in the html file. \n\n-Check if the .js file where you are defining your chart is executing after the definition of the canvas in the html file.\n");
```

Comment:
Chart.js supports multiple platforms, I don't think these tips make any sense to an typescript developer targeting node.js. I think the original error is sufficient.

Reply:
Makes sense, didn't have that in mind. Expect the fix soon.